### DISCUSSION TITLE
Can't get whisper tests to run

### DISCUSSION THREAD

**Discussion by Spungyo** [SYSTRAN/faster-whisper#1103] [Q&A]
I've been trying to start the test files to validate my installation but they never work.

For test_transcribe.py I attempt to run it and nothing happens. Does this mean it has run successfully with no errors? I get no output from the terminal nor can find any kind of file output as far as I'm aware...

**Comment by MahmoudAshraf97** [comment]
how do you run it?

**Reply by Spungyo** [reply]
Sorry I lost access to my account for a bit. I try to run the test python files through CLI.
